### ISSUE TITLE
Percent in Axis label produces hard to read error. need to escape '%' or Warning?

### ISSUE THREAD

**Issue by RichardLettich** [nschloe/matplotlib2tikz#261]
Regarding #50 

When assigning a axis label, I was describing a percent, and used the percent symbol. Everything was produced fine in python, and went as expected.

However, I could not figure out why it kept crashing in latex a week later. It took me around an hour and a half, as the latex error output was not helpful at all

Here's reproducible code

```
import pandas as pd
import matplotlib.pyplot as plt
from matplotlib2tikz import save as tikz_save


#RANDOM DATA
des = [1,2,3,4]
perc_below = [1,2,3,4]
    
    

df3 = pd.DataFrame({'one':perc_below, 'two':des})

df3.plot(fontsize=20, kind='line',y='one',x='two')
plt.xlabel('example',fontsize=20)

# ISSUE LIES BELOW
plt.ylabel('example %',fontsize=20)

tikz_save('example.tex')`



```

Tex output is as expected...

In my opinion, this should be escaped, as the work around is a backslash when assigning the label. However, this would lead to different behavior as matplotlib. (I also export to pngs, which would end up with a backslash in them)

I understand if you do not agree. In that case, a warning when the graph is produced would be great, as again, the latex error was completely unhelpful.

I love the library, BTW!

**Comment by alexdiment** [comment]
I have a similar issue with underscore symbols that need to be manually escaped in the generated tex code.

**Comment by nighthawk9** [comment]
This would be easy to implement, a lot of the LaTeX(ifying) can be done using the matplotlib pgf backend. Currently I have #267 waiting on the author to fix the default environment which gets used. Similarly, the pgf_backend has a function to escape protected characters in LaTeX

**Comment by nighthawk9** [comment]
This issue is solved from PR #267 

I believe this issue can be closed.

**Comment by nschloe** [comment]
Thanks @nighthawk9!
